content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -148,72 +137,52 @@ After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart butt
-   :alt: Ready status for the pick operation while the delivery operation is Waiting Another
-         Operation.
-
-Click on the picking delivery order to begin processing it. If the product is in stock, Odoo will
-automatically reserve the product. Click :guilabel:`Validate` to mark the picking as
-:guilabel:`Done`, then the delivery order will be ready for processing. Since the documents are
-linked, the products which have been previously picked are automatically reserved on the delivery
+Click on the picking (WH/PICK) operation associated with the sales order to reveal the picking
 order.
 
-.. image:: receipts_delivery_two_steps/validate-two-step-pick.png
+Click :guilabel:`Check Availability` on the picking order form. If the product is in stock, Odoo
+automatically reserves the product. Quantities can also be manually set by changing the value in the
+:guilabel:`Quantity` column to match the value in the :guilabel:`Demand` column.
+
+Once ready, click :guilabel:`Validate` to complete the picking and move the product from
+:guilabel:`WH/Stock` to :guilabel:`WH/Output.`
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-picking-form.png
    :align: center
-   :alt: Validate the picking by clicking Validate.
+   :alt: Picking operation form moving product to WH/Output.
 
 Process the delivery
 --------------------
 
-The delivery order will be ready to be processed once the picking is completed, and can be found in
-the :menuselection:`Inventory` application, on the :guilabel:`Inventory Overview` dashboard. Click
-the :guilabel:`# To Process` smart button in the :guilabel:`Delivery Orders` kanban card to begin.
+Once the picking has been processed, the delivery order is generated, and can be processed.
 
-.. tip::
-   The delivery order associated with the :abbr:`SO (Sales Order)` can also be quickly accessed by
-   clicking on the :guilabel:`Delivery` smart button again, and choosing the delivery order on the
-   :guilabel:`Transfers` page (which should now be marked as :guilabel:`Ready`).
+Navigate back to the original sales order by going to :menuselection:`Sales app`, and selecting the
+associated sales order.
 
-.. image:: receipts_delivery_two_steps/deliver-two-step-kanban.png
-   :align: center
-   :alt: The delivery order can be seen in the Inventory Kanban view.
+The :guilabel:`Delivery` smart button now indicates there are two transfers. Click the
+:guilabel:`Delivery` smart button to reveal each transfer tied to this sales order: the completed
+picking, and the new delivery (WH/OUT) operation. Click the delivery operation to open the delivery
+order.
 
-Click on the delivery order associated with the :abbr:`SO (Sales Order)`, then click on
-:guilabel:`Validate` to complete the move.
+Once ready, click :guilabel:`Validate` to complete the delivery.
 
-.. image:: receipts_delivery_two_steps/validate-two-step-delivery.png
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-order.png
```

Comment:
Too zoomed in, would include the larger form/UI

Reply:
Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅